### DISCUSSION TITLE
Infrastructure monitoring for Windows and other system and integration email alert

### DISCUSSION THREAD

**Discussion by Rohitk0609** [apache/skywalking#9736] [Q&A]
1. Can monitor an infra of windows or other OS. If yes then anyone help me with the configuration or docs.
2. Can we send an alert via email?

**Comment by wu-sheng** [comment]
1. In tech perspective, it can. We don't provide that out of box. If you want to contribute, welcome.
2. No, we don't support that. Use HTTP o gRPC hook to forward notification through email should be easy.

**Reply by Rohitk0609** [reply]
Ok. thanx for your reply
